Services/AvatarService/Sources/LocalStore.swift

```diff
@@ -0,0 +1,34 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import Foundation
+
+struct LocalStore: Sendable {
+    private let documentType = "png"
+    private var fileManager: FileManager { FileManager.default }
+    private let documentDirectory = URL.documentsDirectory
+    
+    // MARK: - Public methods
+
+    public func store(_ data: Data) throws -> String {
+        let documentPath = documentPath(for: UUID().uuidString)
+        try data.write(to: documentPath, options: .atomic)
+        return documentPath.lastPathComponent
+    }
+    
+    public func remove(_ avatar: String) throws {
+        guard fileManager.fileExists(atPath: documentPath(for: avatar).path()) else {
+            return
+        }
+        try fileManager.removeItem(at: documentPath(for: avatar))
+    }
+    
+    // MARK: - Private methods
+    
+    private func documentPath(for id: String) -> URL {
```

Comment:
```suggestion
    private func documentPath(for id: String, documentType: String) -> URL {
```

LocalStore is generic, it should not hardcode `png` inside it, anyone can pass any format